components/dropdown/dropdown-content-mixin.js

```diff
@@ -0,0 +1,556 @@
+import { clearDismissible, setDismissible } from '../../helpers/dismissible.js';
+import { findComposedAncestor, isComposedAncestor } from '../../helpers/dom.js';
+import { getComposedActiveElement, getFirstFocusableDescendant, getPreviousFocusableAncestor } from '../../helpers/focus.js';
+import { html } from 'lit-element/lit-element.js';
+import { styleMap } from 'lit-html/directives/style-map.js';
+
+export const DropdownContentMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			minWidth: {
+				type: Number,
+				reflect: true,
+				attribute: 'min-width'
+			},
+			maxWidth: {
+				type: Number,
+				reflect: true,
+				attribute: 'max-width'
+			},
+			noAutoClose: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-auto-close'
+			},
+			noAutoFit: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-auto-fit'
+			},
+			noAutoFocus: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-auto-focus'
+			},
+			noPadding: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-padding'
+			},
+			noPointer: {
+				type: Boolean,
+				reflect: true,
+				attribute: 'no-pointer'
+			},
+			align: {
+				type: String,
+				reflect: true
+			},
+			boundary: {
+				type: Object,
+			},
+			opened: {
+				type: Boolean,
+				reflect: true
+			},
+			/**
+			 * Private.
+			 */
+			openedAbove: {

```

Comment:
I think this should become `_openedAbove` since it is private.  I'd probably also reflect as `_opened-above` so there's no confusion if anyone if looking at the HTML. If updating the attribute, don't forget to update the styles that reference it.

Reply:
Talked with @dbatiste , `opened-above` is referenced in the LMS context menu CSS so going to leave this name the same to prevent introducing a breaking change.